src/sentry/event_manager.py

```diff
@@ -870,7 +870,7 @@ def _handle_regression(self, group, event, release):
             # we only mark it as a regression if the event's release is newer than
             # the release which we originally marked this as resolved
             has_resolution = GroupResolution.objects.filter(
-                Q(release__date_added__gt=release.date_added) | Q(release=release),
+                Q(release__date_added__gt=release.date_added) | Q(release=release) | Q(release__isnull=True),
```

Comment:
I could back this change out, but its likely what we'll want to do when we know that theres a commit thats pending that will fix the release.